psycop/projects/scz_bp/model_training/estimator_steps/imputers.py

```diff
@@ -1,6 +1,7 @@
 from typing import Literal
 
 from sklearn.ensemble import RandomForestRegressor
+from sklearn.experimental import enable_iterative_imputer  # noqa
```

Comment:
Why a new edit here?

Reply:
The `IterativeImputer` (used for missforest) is "experimental" which means that one needs to explicitly do the import above for it to work (import `IterativeImputer` is apparently not enough). Got an error when testing the `populate_scz_bp registry` function, but should have been in a different PR - sorry.